EnglishFilter/sections/specific.txt

```diff
@@ -595,6 +595,8 @@ swordz.io##div[style="width:300; height: 250; background-color: #000"]
 disasterscans.com##div[id^="media_image-"].no-icon.heading-style-1
 9tsu.net##.ads-system
 pcworld.com###amazon-bottom-widget
+shiachat.com##body > div#ipsLayout_header + center
+||shiachat.com/forum/uploads/monthly_*/*.gif
```

Comment:
It's kinda wide. Make 2 rules.
For example `||shiachat.com/forum/uploads/monthly_*/shiamatch.*.gif` or you can match them via Regex.